js/okx.js

```diff
@@ -3928,13 +3928,15 @@ module.exports = class okx extends Exchange {
          * @method
          * @name okx#fetchLeverage
          * @description fetch the set leverage for a market
+         * @see https://www.okx.com/docs-v5/en/#rest-api-account-get-leverage
          * @param {str} symbol unified market symbol
          * @param {dict} params extra parameters specific to the okx api endpoint
          * @returns {dict} a [leverage structure]{@link https://docs.ccxt.com/en/latest/manual.html#leverage-structure}
          */
         await this.loadMarkets ();
-        const marginMode = this.safeStringLower (params, 'mgnMode');
-        params = this.omit (params, [ 'mgnMode' ]);
+        const defaultMarginMode = this.safeString2 (this.options, 'defaultMarginMode', 'marginMode', 'cross');
+        const marginMode = this.safeString2 (params, 'mgnMode', 'marginMode', defaultMarginMode);
+        params = this.omit (params, [ 'mgnMode', 'marginMode' ]);
```

Comment:
We don't have to omit `mgnMode` in this case, cause it must be propagated to the exchange:

```suggestion
        params = this.omit (params, 'marginMode');
```